llvm/test/Transforms/InstCombine/store.ll

```diff
@@ -345,6 +345,22 @@ define void @store_to_readonly_noalias(ptr readonly noalias %0) {
   ret void
 }
 
+; if ptr which store to is addrspacecast, and its target is alloca. remove it.
+define void @src_store_oneuse_addrspaceast_alloca(ptr align 8 %arg) {
```

Comment:
Should run with the max iterations flag to make sure that problem is fixed?

Reply:
Is this patch not enough to fix this issue? I thought it would be enough to solve the problem of `reduce-instcombine-fixpoint-error.ll.txt`.

Reply:
I mean I don't remember the exact iteration status. Without this patch, and without additional pass arguments, do you see the verifier error in this test as-is?

Reply:
I see. I was worried that I might have misunderstood and written a patch.

Reply:
I just checked. I see the fix point error without any flags, so it's fine.